pygmt/helpers/decorators.py

```diff
@@ -102,23 +102,129 @@
             Control how aspatial data are handled during input and output.
             Full documentation is at :gmt-docs:`gmt.html#aspatial-full`.
          """,
+    "b": r"""
+        binary : bool or str
+            **i**\|\ **o**\ [*ncols*][*type*][**w**][**+l**\|\ **b**]
```

Comment:
Need a fullstop here otherwise the rendering looks a bit weird.
```suggestion
            **i**\|\ **o**\ [*ncols*][*type*][**w**][**+l**\|\ **b**].
```
![image](https://user-images.githubusercontent.com/23487320/128098153-9e8ebcf1-3ab2-4914-aeb7-7a5cf4c4a039.png)